platform/web/export/export_plugin.cpp

```diff
@@ -372,25 +373,26 @@ bool EditorExportPlatformWeb::has_valid_export_configuration(const Ref<EditorExp
 	String err;
 	bool valid = false;
 	bool extensions = (bool)p_preset->get("variant/extensions_support");
+	bool compatibility_mode = (bool)p_preset->get("variant/compatibility_mode");
 
 	// Look for export templates (first official, and if defined custom templates).
-	bool dvalid = exists_export_template(_get_template_name(extensions, true), &err);
-	bool rvalid = exists_export_template(_get_template_name(extensions, false), &err);
+	bool debug_standard_valid = exists_export_template(_get_template_name(extensions, compatibility_mode, true), &err);
```

Comment:
That rename sounds like you were planning to duplicate the checks before adding it as a bool to the function. So the names can likely stay `dvalid` and `rvalid` like they are for all other platforms.